### PR TITLE
implementing storing chest data into Manager.Screens in bot/screen

### PR DESCRIPTION
**Pull request by qwqtoday**

### CODE REVIEW

**Review by Tnze — APPROVED** [review]

File: bot/msg/chat.go [hunk 1]
```diff
@@ -175,6 +175,30 @@ func (m *Manager) SendMessage(msg string) error {
...
+	if err := binary.Read(rand.Reader, binary.BigEndian, &salt); err != nil {
+		return err
+	}
+
+	err := m.c.Conn.WritePacket(pk.Marshal(
+		packetid.ServerboundChatCommand,
+		pk.String(command),
+		pk.Long(time.Now().UnixMilli()),
+		pk.Long(salt),
+		pk.Ary[pk.VarInt]{Ary: []pk.Tuple{}},
```
Comment (Tnze) [inline bot/msg/chat.go:194]: Should we support argument signatures?

Comment (qwqtoday) [reply bot/msg/chat.go:194]: Maybe but we didn’t support signatures for sending messages anyways

**Review by Tnze — APPROVED** [review]

**Comment by Tnze** [comment]
Please resolve the conflicts.

**Comment by qwqtoday** [comment]
should be fixed now

**Pull request merged**